src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/OOPTagHelperResolver.cs

```diff
@@ -146,16 +144,5 @@ private static JObject Serialize(ProjectSnapshot snapshot)
 
             return JObject.FromObject(snapshot, serializer);
         }
-
-        private static TagHelperResolutionResult Deserialize(JObject jsonObject)
```

Comment:
@ryanbrandenburg / @tmat I'm getting this locally:

![image](https://user-images.githubusercontent.com/2008729/81993945-a5d8cc00-95fb-11ea-994f-0811516266db.png)

It looks like we still need to provide a deserializer. @tmat is ther a recommended way to do that with the new pattern?

Reply:
I think you'll need to call `remoteClient.TryRunRemoteAsync<JObject>` instead of `remoteClient.TryRunRemoteAsync<TagHelperResolutionResult>` and then Deserialize.